mlir/include/mlir/Dialect/Arith/IR/ArithOpsInterfaces.td

```diff
@@ -106,4 +106,26 @@ def ArithIntegerOverflowFlagsInterface : OpInterface<"ArithIntegerOverflowFlagsI
   ];
 }
 
+def ArithRoundingModeInterface : OpInterface<"ArithRoundingModeInterface"> {
+  let description = [{
+    Access to op rounding mode.
+  }];
+
+  let cppNamespace = "::mlir::arith";
+
+  let methods = [
+    InterfaceMethod<
+      /*desc=*/        "Returns an RoundingModeAttr attribute for the operation",
+      /*returnType=*/  "RoundingModeAttr",
+      /*methodName=*/  "getRoundingModeAttr",
```

Comment:
Do we need the `Attr` suffix on the name?

Reply:
Yes, as that's returning an attribute. That's MLIR convention.